dolphinscheduler-bom/pom.xml

```diff
@@ -895,6 +896,12 @@
                 <artifactId>protobuf-java</artifactId>
                 <version>${protobuf.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.huaweicloud</groupId>
+                <artifactId>esdk-obs-java-bundle</artifactId>
+                <version>${esdk-obs.version}</version>
+            </dependency>
```

Comment:
please add the license of your 3-party dependency
FYI: https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/DS-License.md